### PR TITLE
Fix: DS Custom Icons

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- replace custom app icon with hero icon
- update connect button ux
- update stories

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: libs/design-system-components/src/utils/iconByAppType.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (SeverS) [inline libs/design-system-components/src/utils/iconByAppType.tsx]: iconByAppType is actually a component so no need to put it in the utils. Maybe create a generic component like ExtensionIcon and also add a story

Comment (josenriagu) [reply libs/design-system-components/src/utils/iconByAppType.tsx]: Aha, yyea. Will do that now

Comment (SeverS) [reply libs/design-system-components/src/utils/iconByAppType.tsx]: another question: why is this not in the ds-core?

Comment (josenriagu) [reply libs/design-system-components/src/utils/iconByAppType.tsx]: Most of the components requiring it were in DS components. That's why I kept it there

Comment (SeverS) [reply libs/design-system-components/src/utils/iconByAppType.tsx]: I think it will be best to move it to ds-core - adjust typings

Comment (josenriagu) [reply libs/design-system-components/src/utils/iconByAppType.tsx]: alright, that works

**Review by SeverS — APPROVED** [review]

**Review by quininez — APPROVED** [review]

**Pull request merged**